src/components/UpcomingEvents.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Link } from 'gatsby'
+
+const UpcomingEvents = () => {
+  return (
+    <section>
+      <h1>No Upcoming Events</h1>
+      <p>
+        Join our <Link to="">Meeting group</Link> to stay up to date on what's
+        coming up.
+      </p>
+      <button>
+        <Link to="">Join our Meetup group</Link>
```

Comment:
Let's make this an external link to [https://www.meetup.com/code-for-chicago/?action=join&_cookie-check=mMhgrIfroDbiH7dR](https://www.meetup.com/code-for-chicago/?action=join&_cookie-check=mMhgrIfroDbiH7dR)